src/app/components/ThemeProviderSCSSModules/fontFaces.module.scss

```diff
@@ -0,0 +1,223 @@
+// fontFaces.module.scss
+// SCSS mixins for @font-face rules, matching the fontFaces.ts constants
+// Usage: @include reith-serif-regular-face; etc.
```

Comment:
seems a pretty direct conversion from the ts constants: https://github.com/bbc/simorgh/blob/latest/src/app/components/ThemeProvider/fontFaces.ts